galaxy_importer/collection.py

```diff
@@ -56,6 +57,10 @@ def _import_collection(file, filename, logger):
             pkg_tar.extractall(extract_dir)
         data = CollectionLoader(extract_dir, filename, logger=logger).load()
 
+    ansible_test_runner = runners.get_runner()
```

Comment:
a config.Config() instance could be passed into import_collection() and passed to runners.get_runner().

That would potentially allow each collection import to use a different test runner config/context.